src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -158,12 +158,10 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
             // The Trust Details Validator will only check the 'ceased date' if full validation is being performed
             trustDetailsValidator.validate(overseasEntitySubmissionDto, errors, loggingContext, isFullValidation);
 
-            if (!overseasEntitySubmissionDto.isForUpdateOrRemove()) {
-                // Note that this validation is only done for registrations
-                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-                historicalBeneficialOwnerValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-                trustCorporateValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
-            }
+            trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
```

Comment:
As well as Postman testing and checking unit-tests pass, have you run through with Docker/Tilt, doing an update and remove filing for an OE with a trust (OE001022) and got to the confirmation screen on both journeys?

Reply:
We would need to implement https://companieshouse.atlassian.net/browse/UAR-1536 before that can be made possible with respect to testing corporate entities linked to trusts, but have run basic update and remove journeys to ensure nothing is broken.

Reply:
Tested everything I can test.

Reply:
What does that mean Matt? Have you tested end to end Update and Remove journeys with Docker/Tilt?

Reply:
![Screenshot 2024-06-07 at 14 27 27 (2)](https://github.com/companieshouse/overseas-entities-api/assets/38655987/afbc3901-db37-4740-a6c5-de880c293fc6)
![Screenshot 2024-06-07 at 14 37 30 (2)](https://github.com/companieshouse/overseas-entities-api/assets/38655987/17437258-f5c3-45ea-abb5-ef4c194980b2)

Yes checked both update and remove twice each time with different journeys change with individual trust and no change.

Reply:
When I test with the OE web 'main' branch and this API branch I get an API validation error when moving off this page:

![Screenshot 2024-06-11 at 09 14 03](https://github.com/companieshouse/overseas-entities-api/assets/21172560/09604eac-8385-4e6d-bc16-dc26ce1c5647)

Reported errors are:

`{"created":"2024-06-11T08:18:20.659Z","event":"error","namespace":"overseas-entities-api","context":"BTg8ame1YuXDuaPsmcZwUOX9cBWm","data":{"transaction_id":"154792-565117-180938","message":"Validation errors : {\"errs\":[{\"error\":\"trusts.CORPORATE.is_corporate_still_involved_in_trust must not be null\",\"location\":\"trusts.CORPORATE.is_corporate_still_involved_in_trust\",\"locationType\":\"request-body\",\"type\":\"ch:validation\"},{\"error\":\"trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address must not be null\",\"location\":\"trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address\",\"locationType\":\"request-body\",\"type\":\"ch:validation\"}]}","overseas_entity_id":"666807ea53fcf84f3e2ecb78"}}`

Do you not get the same @mattch1?

I think the `trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address must not be null` is unrelated to this change (a recently observed issue) but `trusts.CORPORATE.is_corporate_still_involved_in_trust must not be null` is a problem and I think we need to raise as this validation will kick in and the user is not forced to go into these screens on an Update/Remove journey and answer the new question...

Reply:
Just run this @mwestacott and got the same: 

{"created":"2024-06-11T10:29:26.489Z","event":"info","namespace":"overseas-entities-api","context":"B7tMutFBi3oEUaHeWo6TfSPCr5lr","data":{"message":"trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address Field is null"}}
{"created":"2024-06-11T10:29:26.490Z","event":"info","namespace":"overseas-entities-api","context":"B7tMutFBi3oEUaHeWo6TfSPCr5lr","data":{"message":"trusts.CORPORATE.is_corporate_still_involved_in_trust Field is null"}}
{"created":"2024-06-11T10:29:26.509Z","event":"error","namespace":"overseas-entities-api","context":"B7tMutFBi3oEUaHeWo6TfSPCr5lr","data":{"transaction_id":"114617-057617-181012","message":"Validation errors : {\"errs\":[{\"error\":\"trusts.CORPORATE.is_corporate_still_involved_in_trust must not be null\",\"location\":\"trusts.CORPORATE.is_corporate_still_involved_in_trust\",\"locationType\":\"request-body\",\"type\":\"ch:validation\"},{\"error\":\"trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address must not be null\",\"location\":\"trusts.INDIVIDUAL.is_service_address_same_as_usual_residential_address\",\"locationType\":\"request-body\",\"type\":\"ch:validation\"}]}","overseas_entity_id":"6668252e2ae4db2f90ce68c2"}}

Reply:
The question now is what data will need to have been submitted before this check can be made?

Reply:
If we are trying to change existing data that was formed without these fields, which it will be then send it back into the api, then it may be this that is forcing the question to be answered.

Reply:
It seems like we might just have to allow a 'null' value for this new field, as user isn't forced into answering the question. If 'false' then the ceased date checks would run. But... nothing decided, so think we have to wait for Vern's new story and a mini-refinement to take place. Maybe add a flag to your JIRA story for this?

Reply:
Yes, null values now being allowed so resolving this comment.